src/locales/us/en/index.json

```diff
@@ -3,8 +3,15 @@
   "dontHaveAnAccount": "Don’t have an account? <a href='/register'>Sign up here!</a>",
   "closeModal": "Close Modal",
   "chooseDifferentUsername": "Please choose a different username.",
-  "ensurePasswordIsMoreThan": "Ensure password contains more than {minPasswordLength} characters.",
+  "constraintViolationForPassword": "Your password does not meet the acceptance criteria.",
```

Comment:
Lot's of strings!!!! I need to go through all of these and clean them up at some time.